### PR TITLE
Change branch trigger to main

### PR DESCRIPTION
**Pull request by zkoppert**
@Chocrates How are the version numbers decided for a new release? 

I don't think this was working before since we are on master branch.

### CODE REVIEW

**Comment by zkoppert** [comment]
@Chocrates If I understand you correctly, the instructions for a new release would be those below. Does that sound correct? If so, I will throw them in the project documentation.

1. Create a branch from the master branch `git checkout master; git checkout -b release-x.x`
1. Run locally `lerna version`
1. Select the correct next version _ie. v2.0.1_
1. Commit changed files with the exact commit message `chore(release): lerna publish`
1. Push these changes `git push`
1. Open a pull request, get approval, and merge
1. Once these changes are merged into `master` branch, automation will [create a release](https://github.com/github/github-artifact-exporter/blob/main/.github/workflows/release.yml), [build the artifacts, and upload them to the release](https://github.com/github/github-artifact-exporter/blob/main/.github/workflows/build.yml)

**Comment by Chocrates** [comment]
@zkoppert nope.  
I haven't tested from a different branch, but simply running `lerna publish` from main will tag the current sha as a new release and push the bits to GPR and the tag up to origin.  
The workflows should notice the new tag and kick off the release automation.

**Review by Chocrates — APPROVED** [review]
💠

**Pull request merged**
